0000-template.md

```diff
@@ -1,6 +1,7 @@
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: (leave this empty)
-- Ember Issue: (leave this empty)
+- Relevant Team(s): (fill this in with the team(s) to which this RFC applies)
```

Comment:
```suggestion
- Relevant Team(s): (fill this in with the team(s)–https://emberjs.com/team/–to which this RFC applies)
```

Reply:
Should this link to emberjs.com/team or to the README table as rwjblue suggested?